surreal.js

```diff
@@ -138,8 +166,14 @@ var $ = { // You can use a different name than "$", but you must change the refe
 	//	✂️ Vanilla: document.querySelector(".thing").addEventListener("click", (e) => { alert("clicked") }
 	on(e, name, fn) {
 		if (typeof name !== 'string') return null
-		if ($.isNodeList(e)) e.forEach(_ => { on(_, name, fn) })
-		if ($.isNode(e)) e.addEventListener(name, fn)
+		// Store the current element for use by 'me' in the event handler
+		function listener(evt) {
+			$._evtEl = e
+			try { fn(evt) }
+			finally { delete $._evtEl }
+		}
```

Comment:
No idea if this is the best way to store the current element for an event handler, but it's the only way I could figure out to get `me` working without needing the event as a parameter. I don't like that it doesn't work for vanilla JS registered handlers, but `el(event)` still works in that case.